lib/watir/element_collection.rb

```diff
@@ -40,6 +40,39 @@ def length
     alias_method :size, :length
 
     #
+    # Returns true if no elements are found
+    #
+    # @example
+    #   browser.select_list(name: "new_user_languages").options(class: 'not_here').empty?
+    #   #=> true
+    #
+    # @example
+    #   browser.select_list(name: "new_user_languages").options(id: 'danish').empty?
+    #   #=> false
+    #
+
+    def empty?
+      length == 0
+    end
+
+    #
+    # Returns true if no elements are found
+    #
+    # @example
+    #   browser.select_list(name: "new_user_languages").options(class: 'not_here').exist?
+    #   #=> false
+    #
+    # @example
+    #   browser.select_list(name: "new_user_languages").options(id: 'danish').exist?
+    #   #=> true
+    #
+
+    def exist?
```

Comment:
Isn't it the method that you want? Some method that is the opposite of `#empty?`.